arrow-data-source/common/pom.xml

```diff
@@ -5,7 +5,8 @@
     <parent>
         <artifactId>spark-arrow-datasource</artifactId>
         <groupId>com.intel.oap</groupId>
-        <version>1.1.0</version>
+        <version>1.2.0-snapshot</version>
```

Comment:
@weiting-chen should we use `snapshot `or `SNAPSHOT `here?

Reply:
@weiting-chen will merge first, please modify it if this does not meet our standard